sdk/src/wallet/account/handle.rs

```diff
@@ -46,6 +46,7 @@ pub struct FilterOptions {
 #[derive(Debug, Clone)]
 pub struct AccountHandle {
     account: Arc<RwLock<Account>>,
+    pub(crate) fallback_sync_options: Arc<Mutex<SyncOptions>>,
```

Comment:
This needs to be set on every new start of an application, shouldn't we also persist it?